pom.xml

```diff
@@ -640,12 +644,59 @@
         </plugins>
       </build>
     </profile>
+
     <profile>
       <id>enable-integration-tests</id>
       <properties>
         <skipITs>false</skipITs>
       </properties>
     </profile>
+
+    <profile>
+      <!-- NOTE - this profile is still in progress and needs work -->
+      <id>docFX</id>
+      <activation>
+        <property>
+          <name>docFX</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <version>3.2.0</version>
+            <executions>
+              <execution>
+                <phase>site</phase>
+                <goals>
+                  <goal>aggregate</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <doclet>com.microsoft.doclet.DocFxDoclet</doclet>
+              <docletArtifact>
+                <groupId>com.microsoft</groupId>
+                <artifactId>docfx-doclet</artifactId>
+                <version>1.0-SNAPSHOT</version>
```

Comment:
Is this snapshot version intentional? It's possible to use Sonatype's nightly repositories, but it's not configured by default (we have an `enable-snapshots` profile that sets up the remote Maven repo).

Reply:
I'll put a comment, but this is temporary and NO ONE should be invoking this yet.